base/ltcounts.dtx

```diff
@@ -313,8 +314,20 @@
      \@addtoreset{#1}{@ckpt}%
      \global\expandafter\let\csname p@#1\endcsname\@empty
      \expandafter
```

Comment:
You could move this `\expandafter` after the added comment as  the block starting ` \ifx\csname the#1\endcsname\relax`  looks like a missing `\expandafter`

Reply:
Make that "could move" into a "should move"!

Reply:
done